test/node/download-webseed-torrent.js

```diff
@@ -72,3 +76,46 @@ test('Download using webseed (via .torrent file)', function (t) {
     })
   })
 })
+
+test('Disable webseeds', function (t) {
+  var parsedTorrent = extend(fixtures.leaves.parsedTorrent)
+
+  var httpServer = http.createServer(function (req, res) {
+    t.fail('webseed http server should not get any requests')
+  })
+  var client
+
+  httpServer.on('error', function (err) { t.fail(err) })
+
+  series([
+    function (cb) {
+      httpServer.listen(cb)
+    },
+
+    function (cb) {
+      parsedTorrent.urlList = [
+        'http://localhost:' + httpServer.address().port + '/' + fixtures.leaves.parsedTorrent.name
+      ]
+
+      client = new WebTorrent({ dht: false, tracker: false, webSeeds: false })
+
+      client.on('error', function (err) { t.fail(err) })
+      client.on('warning', function (err) { t.fail(err) })
+
+      client.add(parsedTorrent, {store: MemoryChunkStore})
+
+      // The test above ensures that we can download the whole torrent over webseeds within a
+      // short time. Here, we wait the same amount of time and make sure no HTTP requests happen.
```

Comment:
I'm not sure what this comment means. Since the `webSeeds` option was set to `false`, no download progress will be made, right? This will just timeout after 500ms and move on to the cleanup code below.

That's fine, I think waiting 500ms is a reasonable way to ensure that no http requests get sent to the server, but the comment should be changed to be clearer.